modules/url-titles.js

```diff
@@ -0,0 +1,22 @@
+const cheerio = require('cheerio')
+const request = require('request-promise-cache')
+var getTitle = async function (url) {
+  const response = await request({
```

Comment:
We need a way to ensure that large payloads don't fill your RAM and OOM you off the server.
Perhaps it should only pull the first 1024 bytes of a response?